cmd/commands/datasetCleaner.go

```diff
@@ -75,29 +75,24 @@ For further help see "` + MANUAL + `"`,
 
 		//}
 
+		if devenvFlag {
+			APIServer = DEV_API_SERVER
+			env = "dev"
+		}
 		if testenvFlag {
 			APIServer = TEST_API_SERVER
 			env = "test"
-		} else if devenvFlag {
-			APIServer = DEV_API_SERVER
-			env = "dev"
-		} else {
-			APIServer = PROD_API_SERVER
-			env = "production"
 		}
 
 		color.Set(color.FgRed)
 		log.Printf("You are about to remove a dataset from the === %s === data catalog environment...", env)
 		color.Unset()
 
-		pid := ""
-
-		if len(args) == 1 {
-			pid = args[0]
-		} else {
+		if len(args) < 1 || len(args) > 1 {
```

Comment:
isn't this simply len(args) != 1?

Reply:
it is actually, I was just on auto pilot when writing this code, I'll change it in a bit.